compositor_render/src/transformations/web_renderer/browser_client.rs

```diff
@@ -81,6 +53,61 @@ impl BrowserClient {
             frame_data,
             source_transforms,
             resolution,
+            chromium_thread_sender: None,
+        }
+    }
+
+    pub fn set_chromium_thread_sender(&mut self, sender: Sender<ChromiumSenderMessage>) {
+        self.chromium_thread_sender.replace(sender);
+    }
+
+    fn update_source_transforms(&self, msg: &cef::ProcessMessage) {
+        let mut transforms_matrices = Vec::new();
+        for i in (0..msg.size()).step_by(4) {
+            let Some(x) = msg.read_double(i) else {
+                error!("Expected \"x\" value of DOMRect");
+                continue;
```

Comment:
Is this the correct way to handle an error here?

Reply:
Now that I look at it, I think `return` would be more suitable. There's also a possible issue with synchronization here when the amount of inputs changes